README.md

```diff
@@ -304,6 +305,19 @@ earnings.symbol_id # 4563
 See [#earnings](https://iextrading.com/developer/docs/#earnings) for detailed documentation or [earnings.rb](lib/iex/resources/earnings.rb) for returned fields.
 
 
+### Get Sector Performance
+
+Fetches earnings for a symbol.
+
+```ruby
+earnings = IEX::Resources::Earnings.get('MSFT')
```

Comment:
@gil27 This should be: `sectors = IEX::Resources::Sectors.get('MSFT')`

Reply:
Shame on me 😭